libs/features/pages/content-development/src/lib/main/content-development-page/content-development-page.component.html

```diff
@@ -1 +1,6 @@
-<p>content-development-page works!</p>
+<elewa-website-elewa-content-dev-about-one></elewa-website-elewa-content-dev-about-one>
+
+ <div class="section1">
+   <elewa-website-image-and-text-banner [ImageAndText]="InputData"></elewa-website-image-and-text-banner>
+    </div>
```

Comment:
This part should be moved into the component you just created and called above in line 1

Reply:
@LemmyMwaura  updated changes